### DISCUSSION TITLE
Unofficial VTK Wheels for Python 3.10

### DISCUSSION THREAD

**Discussion by akaszynski** [pyvista/pyvista#2064] [Announcements]
Until Kitware releases official VTK wheels for Python 3.10 on [PyPI](https://pypi.org/project/vtk/), you can use custom wheels PyVista hosted at https://github.com/pyvista/pyvista-wheels for Windows, Linux x64, and MacOS using:

```
pip install --find-links https://wheels.pyvista.org/ pyvista
```

---

If you want to install a specific version of a wheel, you can point directly to the the file itself:

```
pip install https://github.com/pyvista/pyvista-wheels/raw/main/vtk-9.1.0.dev0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
```

---
Also note that you can install vtk via `conda` with for Python 3.10 with:

```
conda install vtk -c conda-forge
```

**Comment by mliza** [comment]
I am following your examples to extract data but it seems that the 'get_point_data' attribute is not present anymore. Has it been renamed? 

 the object has no attribute 'get_point_data'

**Reply by akaszynski** [reply]
This should just be the attribute: [point_data](https://docs.pyvista.org/api/core/_autosummary/pyvista.DataSet.point_data.html#point-data)

**Reply by mliza** [reply]
Thank you for the quick response, I tried point data, and it says: 

VTUIO' object has no attribute 'point_data'

See attached image, on the left is the code and on the right I am trying the point_data and get_point_data.
<img width="1219" alt="Screenshot 2023-03-11 at 1 06 36 AM" src="https://user-images.githubusercontent.com/60142782/224468178-862950c6-e2fb-4bb0-ab12-0f9d616b0bac.png">

**Reply by MatthewFlamm** [reply]
This looks to be in the wrong repo, you are probably looking for https://github.com/joergbuchwald/VTUinterface.  That package doesn't look to use pyvista.
